dapp/src/hooks/useTVL.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  queryKeysFactory,
+  REFETCH_INTERVAL_IN_MILLISECONDS,
+  TOTAL_VALUE_LOCKED_CAP,
+} from "#/constants"
+import { acreApi } from "#/utils"
+import { useQuery } from "@tanstack/react-query"
+
+const { acreKeys } = queryKeysFactory
+
+const useTVL = () => {
+  const { data } = useQuery({
+    queryKey: [...acreKeys.totalAssets()],
```

Comment:
Total assets is a name specific to the [stBTC vault](https://github.com/thesis/acre/blob/main/solidity/contracts/stBTC.sol). But now we include in the TVL not only the stBTC vault total assets, but also deposits made to the [Multi-Asset Vault](https://github.com/thesis/acre/blob/main/solidity/contracts/AcreMultiAssetVault.sol).

Let's replace `totalAssets` with `tvlBtc` across the file. Let's also update the query keys factory.


```suggestion
    queryKey: [...acreKeys.tvlBtc()],
```

Reply:
Refactored for consistency: f4eb4bd1
You called the endpoint "statistics" so I followed the convention.